oops/backplane/pixel.py

```diff
@@ -0,0 +1,70 @@
+################################################################################
+# oops/backplanes/pixel.py: pixel coordinate backplanes
+################################################################################
+
+import oops
+
+from oops.constants import C
+from oops.backplane import Backplane
+
+#===============================================================================
+def radius_in_pixels(self, event_key, radius_in_km=0):
+    """Gridless approximate apparent radius of the body in pixels.
+
+    Input:
+        event_key       key defining the event on the body's path.
+        radius_in_km    If nonzero, override the radius of the body referred to
+                        in the event key.
+    """
+    gridless_key = Backplane.gridless_event_key(event_key)
+    key = ('radius_in_pixels', gridless_key)
+    if key in self.backplanes:
+        return self.get_backplane(key)
+
+    # compute apparent distance
+    event = self.get_surface_event(gridless_key, arrivals=True)
+    distance = event._dep_lt_*C
+
+    # compute apparent enclosing radius
+    if radius_in_km==0:
+        (body, mod) = Backplane.get_body_and_modifier(gridless_key[1])
+        radius_in_km = body.radius
+    radius = radius_in_km/distance / self.obs.fov.uv_scale.values[0]
+
+    return self.register_backplane(key, radius)
+
+#===============================================================================
+def _center_coordinates(self, gridless_key):
```

Comment:
I'm not sure I see the need for this extra function. I suggest you just place these two lines inside the definition of `center_coordinate`.